### PR TITLE
Add list of team members to contact, fixes #71

### LINKED ISSUE #71: Menüpunkt Team
Kontaktvermittlung für alle ÜL. Liste von
- Portrait
- Name
- Telefonnummer
- E-Mail-Adresse

### PR DESCRIPTION
**Pull request by JoeyStk**
Add list of team members to contact, fixes #71

Extend the profile model and display information for users in groups "Team" and "Projektleitung" in "Kontakt".

### CODE REVIEW

**Comment by JoeyStk** [comment]
This push adds the telephone number and an avatar to the user model. Please let me know, if I altered the model correctly or if I should have added it to the foreign key. 
Should the image and the telephone number be shown in the template?

File: src/gui/templates/contact.html [hunk 1]
```diff
@@ -18,4 +18,33 @@ <h2>Kontakt</h2>
           <input type="submit" class="ig-button" value="Senden">
         </form>
       </div>
+
+      <h2>Team</h2>
+      {% is_staff %}
+
+        {% for user in author %}
+        test
```
Comment (svenseeberg) [inline src/gui/templates/contact.html:26]: The `test` is probably not required anymore? ;-)

File: src/gui/templates/contact.html [hunk 2]
```diff
@@ -18,4 +18,33 @@ <h2>Kontakt</h2>
           <input type="submit" class="ig-button" value="Senden">
         </form>
       </div>
+
+      <h2>Team</h2>
+      {% is_staff %}
```
Comment (svenseeberg) [inline src/gui/templates/contact.html:23]: Is `is_staff` defined anywhere? You probably need to iterate over all users and then check for the user attribute `is_staff`.

For example like this:
```
{% for user in author %}
{% if user.is_staff %}

render some HTML

{% endif %}
{% endfor %}
```

File: src/gui/templates/contact.html [hunk 3]
```diff
@@ -18,4 +18,33 @@ <h2>Kontakt</h2>
...
+
+        {% for user in author %}
```
Comment (svenseeberg) [inline src/gui/templates/contact.html:25]: What is `author`? I think you need to set the variable in the view first. That means that line https://github.com/digitalfabrik/ish-goalkeeper/blob/master/src/gui/views/contact.py#L16 needs a `context` being set as in https://github.com/digitalfabrik/ish-goalkeeper/blob/master/src/gui/views/contact.py#L32. The context needs to contain a list of existing users, over which you can then iterate in the template.

**Review by svenseeberg — CHANGES_REQUESTED** [review]
This pull request still needs some features before it is complete.

File: src/gui/templates/contact.html [hunk 4]
```diff
@@ -1,21 +1,44 @@
...
+      {{ user.telephone }}
+    </li>
+    {% endfor %}
+</ul>
+
+<h2>Team</h2>
+<ul>
+  {% for user in team %}
+    <li>
+      {{ user.avatar }}
```
Comment (svenseeberg) [inline src/gui/templates/contact.html:37]: Does it make sense to do a little more styling here? In this case we have everything in one line, right?

**Pull request merged**